File: src/assets/usage-examples-code/utilities/to_lowercase/to_lowercase-1-basic-usage.cpp
#include <iostream>
#include "splashkit.h"

int main()
{
    std::string inputText;
    std::cout << "Enter a text: ";
    std::getline(std::cin, inputText); // Read user input
    std::string lowerCaseText = to_lowercase(inputText);
    std::cout << "Original text: " << inputText << std::endl;
    std::cout << "Lowercase text: " << lowerCaseText << std::endl;
    return 0;
}

File: src/assets/usage-examples-code/utilities/to_lowercase/to_lowercase-1-basic-usage.cs
using SplashKitSDK;
using System;

class Program
{
    static void Main(string[] args)
    {
        Console.Write("Enter a text: ");
        string originalText = Console.ReadLine();
        string lowercaseText = SplashKit.ToLowercase(originalText);
        Console.WriteLine("Original Text: " + originalText);
        Console.WriteLine("Lowercase Text: " + lowercaseText);
    }
}
